server/tests/integration/test_sudo.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+"""
+Tests for the permissions system
+"""
+
+import os
+os.environ['FLASK_CONF'] = 'TEST'
+import datetime
+import urllib
+import flask
+
+from test_base import APIBaseTestCase, unittest #pylint: disable=relative-import
+
+from app import models, utils
+
+from google.appengine.ext import ndb
+
+from app import auth
+from app.authenticator import Authenticator, AuthenticationException
+
+class SudoUnitTest(APIBaseTestCase):
+    def get_accounts(self):
+        """
+        Returns the accounts you want to exist in your system.
+        """
+        return {
+            "student0": models.User(
+                email=["dummy@student.com"]
+            ),
+            "student1": models.User(
+                email=["other@student.com"]
+            ),
+            "staff": models.User(
+                email=["dummy@staff.com"]
+            ),
+            "admin": models.User(
+                email=["dummy@admin.com"],
+                is_admin=True
+            ),
+            "dummy_admin": models.User(
+                email=["dummy3@admin.com"],
+                is_admin=True
+            )
+        }
+
+    def get(self, url, *args, **kwds):
+        """
+        Makes a get request.
+        """
+        url = url
```

Comment:
That line shouldn't be there. I brought in the get function because this test shouldn't be making json requests.